### ISSUE TITLE
False positive for shorthand_operator

### ISSUE THREAD

**Issue by fpg1503** [realm/SwiftLint#1163]
I get two false positives for the `shorthand_operator` rule in the following block:

```swift
if targetOffset > currentOffset {
    page = ceilf(currentOffset / pageWidth) //here
} else {
    page = max(floorf(currentOffset / pageWidth), 0) //and here
}
```

I couldn't find any related issues.

**Comment by marcelofabri** [comment]
I think this example was accidentally fixed in #1157, but we still should look at it when it's a commutative operator.
